aea/multiplexer.py

```diff
@@ -159,7 +161,6 @@ def __init__(
         self._send_loop_task = None  # type: Optional[asyncio.Task]
 
         self._loop: asyncio.AbstractEventLoop = loop if loop is not None else asyncio.new_event_loop()
-        self._lock: asyncio.Lock = asyncio.Lock(loop=self._loop)
```

Comment:
Why the removal of the lock?

Reply:
At a minimum init should specify the variable as None; defining new instance variables elsewhere is not recommended.